apps/api-server/src/lib/mail-config.js

```diff
@@ -3,54 +3,62 @@ const config = require('config');
 const configField = require('../models/lib/config-field');
```

Comment:
Enige wat er in mail-config is bijgekomen is formatting, is niet relevant om hiernaar te kijken